documentation/typography.stories.mdx

```diff
@@ -22,15 +22,7 @@ We gebruiken de Lucida font family omdat het een font is dat goed leesbaar is op
 
 ### Fallback fonts
 
-`TODO: Bespreken met huisstijl wat het fallback font in ons design system zou moeten zijn.`
-
-Ons fallback font is: Verdana.
-
-### Arial
-
-`TODO: Bespreken met huisstijl wat het fallback font in ons design system zou moeten zijn.`
-
-Het Arial font wordt in digitale applicaties niet gebruikt omdat Lucida Grande en de fallback fonts beter leesbaar zijn op digitale applicaties.
+Ons fallback font is: Arial.
```

Comment:
Als Lucida niet de fallback font is, dan moet dit ook aangepast worden in de design tokens:
```json
{
  "utrecht": {
    "typography": {
      "sans-serif": {
        "font-family": { "value": "\"Lucida Grande\", \"Lucida Sans Unicode\", \"Lucida Sans\", Arial, sans-serif" }
      }
  }
}
```